src/lib/core/breakpoints/break-point-registry.ts

```diff
@@ -75,6 +58,25 @@ export class BreakPointRegistry {
    * for property layoutGtSM.
    */
   get suffixes(): string[] {
-    return this._registry.map(it => !!it.suffix ? it.suffix : '');
+    return this.items.map(it => !!it.suffix ? it.suffix : '');
   }
+
+  /**
+   * Memoized lookup using custom predicate function
+   */
+  private findWithPredicate(key: string, searchFn: (bp: BreakPoint) => boolean) {
+    let response: OptionalBreakPoint = this.findByMap.get(key) || null;
```

Comment:
You don't need `|| null`, `Map.get` returns `undefined`, which will fail the check below. And then you can do `return response || null` if you care about the return type.